src/styles/settings/_colors.scss

```diff
@@ -3,6 +3,7 @@ $thin-air: #caf4ff;
 $white: #ffffff;
 $off-white: #f3f3f3;
 $gray: #cacaca;
+$placeholder-gray: #c2c2c2;
```

Comment:
These are nice changes! I didn't notice this file when writing my changes, but I definitely would have used it if I had.